### DISCUSSION TITLE
Why do I get the Error Module not found; Can't resolve 'reduxjs-toolkit-persist/lib/storage/session

### DISCUSSION THREAD

**Discussion by daheejo** [reduxjs/redux-toolkit#2729] [Q&A] [answered]
Hi friends,
I'd like to use sessionStorage on redux-toolkit
so I imported the module like below.

```
import storageSession from 'reduxjs-toolkit-persist/lib/storage/session';
// store.js
```

But I'm getting the Error `Can't resolve 'reduxjs-toolkit-persist/lib/storage/session`.
I have no idea why I can't import the module though I've installed redux-toolkit.

Anyone can help me?

**Comment by phryneas** [comment, marked as answer]
`'reduxjs-toolkit-persist'` has nothing to do with redux toolkit, it's a third-party library that you need to install separately.

**Reply by daheejo** [reply]
thanks for your answer!
